Doc/using/cmdline.rst

```diff
@@ -426,6 +426,8 @@ Miscellaneous options
    defines the following possible values:
 
    * ``-X faulthandler`` to enable :mod:`faulthandler`;
+   * ``-X oldparser``: enable the traditional LL(1) parser.  See also
+     :envvar:`PYTHONOLDPARSER`.
```

Comment:
Maybe the PEP 617 should be mentioned here. I suggest to directly deprecate this option and announces its removal in 3.10 using a `.. deprecated` markup.

Moreover, the new option should be documented in the ".. versionchanged:: 3.9" below.

Same remark for PYTHONOLDPARSER.